src/main/java/core/basesyntax/service/WriterToCsv.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.service;
```

Comment:
The package name should reflect the structure mentioned in the checklist. Consider moving WriterToCsv to the `service` package if it's not already there, or to `service.impl` if this is an implementation detail.